Bugzilla/Constants.pm

```diff
@@ -666,6 +668,10 @@ use constant EMAIL_LIMIT_EXCEPTION => "email_limit_exceeded\n";
 # (view_job_queue.cgi).
 use constant JOB_QUEUE_VIEW_MAX_JOBS => 2500;
 
+# Maximum number of times an email can bounce for an account
+# before the account is completely disabled.
+use constant BOUNCE_COUNT_MAX => 5;
```

Comment:
We also need a `BOUNCE_REACTIVATION_MAX` count, and track per-user how many times they have reactivated delivery after bounce-triggered disabling.

Currently if someone's email is completely non-deliverable there's nothing stopping them from immediately reactivating bugmail each and every time they hit 5 bounces.  This will increase our SES bounce rates and might lead to Amazon disabling our account.

Once they hit this cap the `/bounced_emails/` page should direct the user to contact the BMO admins.
editusers will need to be updated to allow admins to reset the per-user reactivation counter.

Reply:
> 
> 
> We also need a `BOUNCE_REACTIVATION_MAX` count, and track per-user how many times they have reactivated delivery after bounce-triggered disabling.
> 
> Currently if someone's email is completely non-deliverable there's nothing stopping them from immediately reactivating bugmail each and every time they hit 5 bounces. This will increase our SES bounce rates and might lead to Amazon disabling our account.
> 
> Once they hit this cap the `/bounced_emails/` page should direct the user to contact the BMO admins.
> editusers will need to be updated to allow admins to reset the per-user reactivation counter.

This should not happen as when we set the disabledtext in SES.pm at 5 bounces, the user is automatically logged out the next time they access any page and when they try to login they get the message stating why they are disabled. So they will no longer be able to get to /bounced_emails/ after 5 bounces.

Reply:
Ah right, I misread some code.  `BOUNCE_COUNT_MAX` does exactly what we need :)